dt-assets/js/comments.js

```diff
@@ -113,40 +129,70 @@ jQuery(document).ready(function($) {
+                        commentsSettings.template_dir
+                      }/dt-assets/images/reply.svg">
+                      ${_.escape(commentsSettings.translations.reply)}
+                  </a>
+              <%}%>
+               <% if (a.user_id === commentsSettings.current_user_id  ) { %>
+                  <% has_Comment_ID = true %>
                   <a class="open-edit-comment" data-id="<%- a.comment_ID %>" data-type="<%- a.comment_type %>" style="margin-right:5px">
-                      <img src="${commentsSettings.template_dir}/dt-assets/images/edit-blue.svg">
+                      <img src="${
+                        commentsSettings.template_dir
+                      }/dt-assets/images/edit-blue.svg">
                       ${_.escape(commentsSettings.translations.edit)}
                   </a>
                   <a class="open-delete-comment" data-id="<%- a.comment_ID %>">
-                      <img src="${commentsSettings.template_dir}/dt-assets/images/trash-blue.svg">
+                      <img src="${
+                        commentsSettings.template_dir
+                      }/dt-assets/images/trash-blue.svg">
                       ${_.escape(commentsSettings.translations.delete)}
                   </a>
-               <% } %>
-            </p>
+             <%}%>
+          </p>
+            <div class="panel panel-default panel-comment-<%- a.comment_ID%> panel-collapsed" style="display:none">
+             <div class="panel-body">
+
+
+                     <textarea class="form-control input-lg reply-textarea" class="mention" id="content-<%- a.comment_ID%>" autofocus placeholder="What do you want to share?"></textarea>
+                     <button class="add-btn button" id="addreply-<%- a.comment_ID%>" data-dismiss="modal" aria-hidden="true" >Submit</button>
+
+
+             </div>
+           </div>
+            <div class= "nested-comments "id="childcomment-<%- a.comment_ID%>"></div>
         <% } else { %>
             <p class="activity-bubble" title="<%- date %>">  <%- a.text %> <% print(a.action) %> </p>
         <%  }
     }); %>
     <% if ( commentsSettings.google_translate_key !== ""  && is_Comment && !has_Comment_ID && activity[0].comment_type !== 'duplicate'
     ) { %>
-        <a class="translate-button showTranslation">${_.escape(commentsSettings.translations.translate)}</a>
-        <a class="translate-button hideTranslation hide">${_.escape(commentsSettings.translations.hide_translation)}</a>
+        <a class="translate-button showTranslation">${_.escape(
+          commentsSettings.translations.translate
+        )}</a>
+        <a class="translate-button hideTranslation hide">${_.escape(
+          commentsSettings.translations.hide_translation
+        )}</a>
         </div>
     <% } %>
     </div>
   </div>`
   )
 
-  $(document).on("click", '.translate-button.showTranslation', function() {
+
+  $(document).on("click",' .translate-button.showTranslation', function() {
```

Comment:
Adding the space in the selector string does not look right.

Reply:
I understand, I was just trying to match the original and I thought it was complaining about not having it. I switched to Visual Studio because of this.